src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -327,7 +327,8 @@ class CacheEvictionConfig:
         :type aggregation_mode: openvino_genai.AggregationMode
     """
```

Comment:
please, also update docs for `apply_rotation` on pybind C++ side

Reply:
Done